tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -298,10 +298,19 @@ public ProcessResult RunSampleAndWaitForExit(MockTracerAgent agent, string argum
 
             var newConfig = Path.GetTempFileName();
 
+            var virtualAppSection = subAppPath switch
+            {
+                null or "" or "/" => "/",
```

Comment:
This looks to be the default case when the VirtualApplicationPath is not set. How does it work when we replace `[VIRTUAL_APPLICATION]` with `"/"`? That seems like it would break the XML and that we should instead use an empty string. Can you walk me through that?

Reply:
You're absolutely right 🤦‍♂️ This is a leftover from when I thought I could _just_ change the `path` attribute for the app pool to be either `/` or `/myapp`. That doesn't work though - if you have a virtual app then you have to have _both_ the `/` app and the `/myapp` app, hence changing it to add the extra section. This should be `""` as you suspected. 

Luckily IISExpress is apparently smart enough to ignore my gaff, so the fix in https://github.com/DataDog/dd-trace-dotnet/pull/2851/commits/5ec0eea5483d97f3a5043add3d4df0978eec92d5 shows nothing is broken